interface/billing/new_payment.php

```diff
@@ -524,6 +539,14 @@ function FillUnappliedAmount()
     $('select').removeClass('class1 text')
 });
 </script>
+<script>
+<?php
+    // jQuery script to change state dynamically
+    $user_settings_php_path = '../../library/ajax/user_settings.php';
+    $arr_files = '"new_payment", "search_payments", "era_payments"'; // the linked files that need to have same state, if none comment out line
```

Comment:
Couldn't we just use the array above with these files already and iterate through them in the expand_contract_js.php script?

Reply:
:)